lib/modules/datasource/types.ts

```diff
@@ -21,6 +21,11 @@ export interface DigestConfig {
 export interface GetReleasesConfig {
   packageName: string;
   registryUrl?: string;
+  filter?: ReleaseFilter;
```

Comment:
To be discussed: how to pass filter for `git ls-remote` through `getReleases` interface